llvm/test/CodeGen/DirectX/ContainerData/RootSignature-Parameters.ll

```diff
@@ -21,34 +21,34 @@ attributes #0 = { "hlsl.numthreads"="1,1,1" "hlsl.shader"="compute" }
 !9 = !{ !"UAV", i32 5, i32 1, i32 10, i32 5, i32 2 }
 
 ;CHECK-LABEL: Definition for 'main':
-;CHECK-NEXT:  Flags: 0x000001
-;CHECK-NEXT:  Version: 2
-;CHECK-NEXT:  RootParametersOffset: 24
-;CHECK-NEXT:  NumParameters: 3
-;CHECK-NEXT:   - Parameter Type: 1
-;CHECK-NEXT:     Shader Visibility: 0
-;CHECK-NEXT:     Register Space: 2
-;CHECK-NEXT:     Shader Register: 1
-;CHECK-NEXT:     Num 32 Bit Values: 3
-;CHECK-NEXT:   - Parameter Type: 3
-;CHECK-NEXT:     Shader Visibility: 1
-;CHECK-NEXT:     Register Space: 5
-;CHECK-NEXT:     Shader Register: 4
+;CHECK-NEXT: Flags: 0x000001
+;CHECK-NEXT: Version: 2
+;CHECK-NEXT: RootParametersOffset: 24
+;CHECK-NEXT: NumParameters: 3
+;CHECK-NEXT: - Parameter Type: Constants32Bit
+;CHECK-NEXT:   Shader Visibility: All
```

Comment:
Try not to change the whitespace here, it makes it harder to tell what's changed.